### PR TITLE
sprints: add schedule and links

### PR DESCRIPTION
**Pull request by cmaureir**
- Adding the schedule section for the sprint day
- Adding "buttons" to the header banner for quick access
- Adding documentation icon for the projects descriptions

### CODE REVIEW

**Comment by cmaureir** [comment]
Feedback is welcome once the build is ready to preview ;)

File: pages/sprint.vue [hunk 1]
```diff
@@ -53,6 +61,53 @@
...
+
+            Instructions to join our Mattermost server and to watch the transmission on YouTube will be send by email!
+            </p>
+            <table>
+                <tr>
+                    <td><b>Time UTC</b> <a style="color: gray;" href="https://time.is/UTC">check it now</a>!</td>
+                    <td><b>Activity</b></td>
+                </tr>
+                <tr>
+                    <td style="text-align: center;">07:00h</td>
```
Comment (jtpio) [inline pages/sprint.vue:94]: Wondering if the "h" at the end is needed?

**Review by jtpio — APPROVED** [review]
Looks great, thanks!

File: pages/sprint.vue (hunk 1 above)
Comment (cmaureir) [reply pages/sprint.vue:94]: maybe I can add `a.m.` or `p.m.` for consistency with the conference schedule

File: pages/sprint.vue [hunk 2]
```diff
@@ -53,6 +61,53 @@
...
+                <tr>
+                    <td style="text-align: center;">15:00h</td>
+                    <td>Wrap-up (Block A)</td>
+                </tr>
+                <tr>
+                    <td style="text-align: center;">16:00h</td>
+                    <td>Kick-off (Block B)</td>
+                </tr>
+                <tr>
+                    <td style="text-align: center;">23:59h</td>
```
Comment (cmaureir) [inline pages/sprint.vue:106]:
```suggestion
                    <td style="text-align: center;">07:00 a.m.</td>
                    <td>Kick-off (Block A)</td>
                </tr>
                <tr>
                    <td style="text-align: center;">3:00 p.m.</td>
                    <td>Wrap-up (Block A)</td>
                </tr>
                <tr>
                    <td style="text-align: center;">4:00 p.m.</td>
                    <td>Kick-off (Block B)</td>
                </tr>
                <tr>
                    <td style="text-align: center;">11:59 p.m.</td>
```

**Comment by cmaureir** [comment]
let's see how does it look with the am/pm

**Comment by cmaureir** [comment]
Screenshot_2020-10-15 JupyterCon2020

:+1: or :-1: ?

**Comment by jtpio** [comment]
Looks good, and it's the same format as for the talks schedule.

**Comment by jtpio** [comment]
Thanks!

**Pull request merged**